From aaae23c228cfc5fd60923137bbfe95fbb19ed039 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Emilio=20L=C3=B3pez?= <turl@tuxfamily.org>
Date: Sat, 18 Aug 2012 06:23:49 +0000
Subject: [PATCH 827/944] ump: fix build error with parallel make

Some UMP components expect the Mali module to be built already,
so we should add a dependency to indicate it. Otherwise build
fails with missing files.
---
 drivers/gpu/mali/ump/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/mali/ump/Kconfig b/drivers/gpu/mali/ump/Kconfig
index ecf5435..2277a53 100644
--- a/drivers/gpu/mali/ump/Kconfig
+++ b/drivers/gpu/mali/ump/Kconfig
@@ -1,6 +1,6 @@
 config UMP
 	tristate "UMP support"
-	depends on ARM
+	depends on ARM && MALI400
 	---help---
 	  This enables support for the UMP memory allocation and sharing API.
 
-- 
1.8.0

